src/sig.cpp

```diff
@@ -289,7 +289,9 @@ size_t getNumBlockLevels(const SIG &sig) {
 LoopVars LoopVars::create(const SIG &sig, const Storage &storage) {
   class LoopVarsBuilder : protected SIGVisitor {
   public:
-    LoopVarsBuilder(const Storage &storage) : storage(storage) {}
+    LoopVarsBuilder(const Storage &storage)
+        : storage(storage), numBlockLevels(0), currBlockLevel(0) {
```

Comment:
I get a compiler warning here asking me to define `currBlockLevels` before `numBlockLevels` (since this is the order they are defined in in the class definition.